src/doc/DocAnnotator.js

```diff
@@ -396,6 +397,7 @@ class DocAnnotator extends Annotator {
         // Highlight listeners on desktop & mobile
         if (this.plainHighlightEnabled || this.commentHighlightEnabled) {
             this.annotatedElement.addEventListener('mouseup', this.highlightMouseupHandler);
+            this.annotatedElement.addEventListener('wheel', this.resetHighlightSelection);
```

Comment:
Does wheel also fire if you are scrolling through overflowed content in a dialog? That might clear things?

Reply:
i'll add a check to only reset the highlight if the mouse is not over an annotation dialog